### ISSUE TITLE
Add ability to share to Reddit app

### ISSUE THREAD

**Issue by Nostang3** [puregallery/Pure-Gallery-Support#27]
When trying to share a photo from the app my reddit app(redditisfun) doesn't show up in the list. If I open up Quickpik the option is there. Honestly not sure if this is a Nougat issue or a coding issue. Just posting to see which.

**Comment by puregallery** [comment]
I can share it to redditisfun though...anyway, since it can be accessed through Quickpik, there's definitely something missing here. Will look into this.

**Comment by Nostang3** [comment]
I tried uninstalling RiF and reinstalling just in case. I know sometimes restoring from TB+ will mess things up but I wasn't sure if I restored it via that or installed manually. Either way, a uninstall, reboot, and install did not add RiF to the share via menu.

**Comment by puregallery** [comment]
Was testing this on Quickpic and they actually use a custom share menu and not the default Android's share option. Guess I need to implement it as well...

**Comment by Nostang3** [comment]
Ah, that explains it. I guess RiF doesn't broadcast the right intent for a normal gallery app to pick it up. 

I also saw you pushed an update. Are the finished tags here what's in the build?

**Comment by puregallery** [comment]
I am maintaining both milestones and tags to update the issues here...
The released features/bugsfixes will come up if you search for milestone 0.994 or the released tag. I'll follow the same thing going forward since it is easy to track things.

**Comment by Nostang3** [comment]
Right, that makes sense. Just wanted to check before I started commenting.

**Comment by Nostang3** [comment]
I'm not sure if you added or changed something but I was able to share to Reddit this morning.

**Comment by puregallery** [comment]
Oh, I didn't do anything here...they might have started using the right intent filters.

**Comment by Nostang3** [comment]
Well funny enough it doesn't seem consistent. I tried to do it just now just to test again, and it's not there. Is there some way you can think where it would catch Reddit Is Fun and sometimes it wouldn't so I can try and test it out more?

**Comment by Nostang3** [comment]
I think I figured it out. It seems that when sharing from the album view, e.g. long pressing on the photo to select it, then sharing the RiF app isn't listed as an option. If you go into media view then share, the RiF option is available.

**Comment by Nostang3** [comment]
I think we can close this issue unless you need it for a reminder. You can share to Reddit using the proper steps.

**Comment by puregallery** [comment]
I will look into why selection won't let it share...
